doc/sphinx/Pacemaker_Explained/fencing.rst

```diff
@@ -190,319 +190,316 @@ for ``pacemaker-fenced``.
-   | pcmk_status_timeout  | time    | 60s                | .. index::                             |
-   |                      |         |                    |    single: pcmk_status_timeout         |
-   |                      |         |                    |                                        |
-   |                      |         |                    | *Advanced use only.* Specify an        |
-   |                      |         |                    | alternate timeout to use for           |
-   |                      |         |                    | ``status`` actions instead of the      |
-   |                      |         |                    | value of ``stonith-timeout``. Some     |
-   |                      |         |                    | devices need much more or less time to |
-   |                      |         |                    | complete than normal. Use this to      |
-   |                      |         |                    | specify an alternate, device-specific  |
-   |                      |         |                    | timeout.                               |
-   +----------------------+---------+--------------------+----------------------------------------+
-   | pcmk_status_retries  | integer | 2                  | .. index::                             |
-   |                      |         |                    |    single: pcmk_status_retries         |
-   |                      |         |                    |                                        |
-   |                      |         |                    | *Advanced use only.* The maximum       |
-   |                      |         |                    | number of times to retry the           |
-   |                      |         |                    | ``status`` command within the timeout  |
-   |                      |         |                    | period. Some devices do not support    |
-   |                      |         |                    | multiple connections, and operations   |
-   |                      |         |                    | may fail if the device is busy with    |
-   |                      |         |                    | another task, so Pacemaker will        |
-   |                      |         |                    | automatically retry the operation, if  |
-   |                      |         |                    | there is time remaining. Use this      |
-   |                      |         |                    | option to alter the number of times    |
-   |                      |         |                    | Pacemaker retries before giving up.    |
-   +----------------------+---------+--------------------+----------------------------------------+
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Default
+     - Description
+   * - .. _primitive_stonith_timeout:
+
+       .. index::
+          single: stonith-timeout (primitive instance attribute)
+
+       stonith-timeout
+     - :ref:`timeout <timeout>`
```

Comment:
It looks like this one is marked as a timeout, but all timeouts are marked as integers.  Is there a reason for that?

Reply:
That appears to be how we're parsing them

Reply:
Previously, something like pcmk_reboot_timeout was marked as a "time" and its default was "60s".  Now it's an "integer" with a default of "60".

Reply:
Yeah, I was sure I must be missing something, but I could never find where it was treated as anything but an integer number of seconds. The only place I could find it being parsed was https://github.com/ClusterLabs/pacemaker/blob/main/daemons/fenced/fenced_commands.c#L236